daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/Functions.tdml

```diff
@@ -11693,11 +11693,19 @@
     <tdml:document>
       <tdml:documentPart type="byte">00 00 00 01</tdml:documentPart>
     </tdml:document>
-    <tdml:errors>
-      <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>ex:foo</tdml:error>
-      <tdml:error>ambiguous</tdml:error>
-    </tdml:errors>
+    <tdml:infoset>
```

Comment:
@pkatlic: As @stevedlawrence asked, please check these changes to test results carefully.

Reply:
The new test results appear to correct the behavior, with scalarOptional parsing properly and arrayOptional throwing the SDE regarding ambiguous path steps.